docs/topics/db/queries.txt

```diff
@@ -1009,6 +1009,63 @@ Unless you are sure you wish to work with SQL ``NULL`` values, consider setting
 Key, index, and path transforms
 -------------------------------
 
+.. class:: KeyTransform(key_name, *args, **kwargs)
```

Comment:
```suggestion
.. class:: json.KeyTransform(key_name, *args, **kwargs)
```

Reply:
I would move it to the end this paragraph (before the closing notes, warnings, and admonitions).